end2end/tests/reportBuilder.spec.ts

```diff
@@ -2,30 +2,238 @@ import { test, expect } from '@playwright/test';
+
+    //Modify Search Filter using OR
+    await page.getByRole('button', { name: 'Modify Search' }).click();
+    await page.getByRole('row', { name: 'remove filter row AND Current' }).getByLabel('remove filter row').click();
+    await page.getByLabel('add logical or filter').click();
+    await page.getByRole('button', { name: 'Next Step' }).click();
+    await page.getByRole('button', { name: 'Generate Report' }).click();
+    await expect(updateFilter).toBeVisible();
+});
+
+test("Edit report title", async ({ page }) => {
+    await page.goto("https://host.docker.internal/Test_Request_Portal/?a=reports&v=3&query=N4IgLgpgTgtgziAXAbVASwCZJHSAHASQBEQAaEAez2gEMwKpsBeMkGOgYwAtsoI4KAGwBuELOQDmdCNgCCAORIBfUuiyIQHaRIYBPYqyq16jDS3Lsw3bADMGMAPoBWCDQAMAdlZTIcxSBU1bAwIQQhIcUpqKDoGZlZLa0Q3SWk%2FZQBdcgArCjQAOwQUEAK0MDRYqHkaGBksnAYwJGAVEAlwojoaJGQQABZWL3IAZhB6wTQYMqQARjd58gmpsAB5Gxs4cKQ3JSA%3D%3D&indicators=NobwRAlgdgJhDGBDALgewE4EkAiYBcYyEyANgKZgA0YUiAthQVWAM4bL4AMAvpeNHCRosuAtGIQUGZrQb4wzNug54efSLARSR8xPCKooAIQCuyNFBn1GC6kpVr%2BmoRhzyALFbkEAsiZJEAAQk0GSByGQAHhx27Fy8ToLabgQA7F42AEqIcKiKcaoJGknCKWAAzBnyAMrQAObkwaHhUTGsBTwAukA");
+    await page.getByRole('cell', { name: 'TestFormQuery_GroupClickedApprove' }).click();
+    const reportTitle = page.getByLabel('Report Title');
+    await reportTitle.click();
+    await reportTitle.fill('TestForm_NonadminCannotCancelOwnSubmittedRecord_new_Title');
+    await page.getByRole('button', { name: 'Save Change' }).click();
+    const updatedTitle = page.getByRole('cell', { name: 'TestForm_NonadminCannotCancelOwnSubmittedRecord_new_Title' })
+    await expect(updatedTitle).toHaveText('TestForm_NonadminCannotCancelOwnSubmittedRecord_new_Title');
+    await updatedTitle.click();
+    await reportTitle.click();
+    await reportTitle.fill('TestFormQuery_GroupClickedApprove');
+    await page.getByRole('button', { name: 'Save Change' }).click();
+    const revertedTitle = page.getByRole('cell', { name: 'TestFormQuery_GroupClickedApprove' });
+    await expect(revertedTitle).toHaveText("TestFormQuery_GroupClickedApprove");
+});
+
+test("Validate single line textbox", async ({ page }) => {
+    await page.goto("https://host.docker.internal/Test_Request_Portal/?a=reports&v=3&query=N4IgLgpgTgtgziAXAbVASwCZJAVztASQBEQAaEAewAdoBDMCqbAXjJBnoGMALbNgc3oRsAQQByJAL6l0WRCE5D%2BjAJ7E21OgybzW5DmB7YAZoxgB9AKwRaABgDsAodgDyAJRDTZ2DBAA2EJBY5JpQ9IwsbAZGiLbkgpCiEp4AuuQAVhRoAHYIKCD4UABuaJzC5DloYGjhUGK0MMJpBYxgSMDSIPyBRPS0SMggAJxsACxsjuQAzCDNfmgwVUgAjLZr5POLYC7GxvhtsZJAA%3D%3D&indicators=NobwRAlgdgJhDGBDALgewE4EkAiYBcYyEyANgKZgA0YUiAthQVWAM4bL4AMAvpeNHCRosuAizLoAbggrVaDfGGZt0HPDz6RYCFBhyLoxCLvTN5jJdRVqN%2FbUL2iwieEVRQAQgFdkaKGfoLZXYuXjtBE30CAE4AhQIAQRYWCABzKDIYAAIAcXRULwAHYNVQzQEdYSiwABY4iwBZLxIiLJJoMizkMgAPDisQ9TCtCKqnAHZ6xQAlRDhUEpthiocRRQBmKYIAZWhU8jaOrt7%2B1kGeAF0gA%3D");
+    const firstText = page.getByRole("cell", { name: "4331" });
+    await firstText.click();
+    await page.getByLabel('Single line text', { exact: true }).click();
```

Comment:
![image](https://github.com/user-attachments/assets/3e5cb60e-8b99-411d-8117-72a8635a2563)

Please run the entire test suite in order to identify conflicting tests. This is conflicting with `adminPanelFormEditor.spec.ts -> change field heading`.

We can either:
1. Update the older test to restore the original fieldname (this would be preferred, because it resolves a problem in that test)
2. OR update this test to remove `exact: true`

Reply:
fixed